### PR TITLE
Adds tls_validation option to project module and role

### PR DESCRIPTION
**Pull request by Tompage1994**
# What does this PR do?
Adds a missing `tls_validation` option to the project module and role. I've also validated there are currently no mossing API options for other roles (latest AAP 2.4 version)

# How should this be tested?
CI is updated

# Is there a relevant Issue open for this?

resolves #47 

# Other Relevant info, PRs, etc
CI currently broken. #49 will fix it and could be merged first so it can be run through for this PR

### CODE REVIEW

File: roles/project/README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ This also speeds up the overall role.
 |`new_name`|""|no|str|Setting this option will change the existing name (looked up via the name field.)|
 |`description`|""|no|str|Description to use for the Project.|
 |`url`|""|yes|str|A URL to a remote archive, such as a Github Release or a build artifact stored in Artifactory and unpacks it into the project path for use. (Alias: scm_url)|
+|`tls_validation`|""|no|bool|Whether the URL should validate using TLS.|
```
Comment (djdanielsson) [inline roles/project/README.md]: technically isn't the default true?

**Review by djdanielsson — CHANGES_REQUESTED** [review]
I want to double check this one thing and then we are good

File: roles/project/README.md (hunk 1 above)
Comment (Tompage1994) [reply roles/project/README.md]: Yes you're correct. Copy/paste error

**Review by djdanielsson — APPROVED** [review]
LGTM

**Pull request merged**